docs/docs/topics/schema.mdx

```diff
@@ -197,120 +282,127 @@ nodes:
         peer: Example1Car
 ```
 
-### Object Uniqueness
-
-The schema provides 2 mechanisms to enforce the uniqueness of an object, within all the objects of the same kind:
+### Inheritance between generics and nodes
 
-- At the attribute level, with the property `unique`
-- At the Node or Generic level with the property `uniqueness_constraints`
+#### Basic inheritance
 
-Both type of uniqueness constraints will be automatically validated during the following events:
+Inheritance allows Nodes to derive properties from Generics. This relationship ensures that Nodes maintain a consistent set of attributes defined at the Generic level, reducing redundancy and simplifying updates.
 
-- Branch merge
-- Proposed Change
-- Branch rebase
-- Schema update
+#### Single generic inheritance
 
-#### Unique attribute
+When a Node is derived from a single Generic, it inherits all properties defined in that Generic unless overridden at the Node level.
 
-Each attribute with the property `unique: true` will be evaluated individually and will automatically enforce a uniqueness constraint for all the nodes of the same kind.
-If a unique attribute is defined on a Generic, all nodes inheriting from this Generic will respect the uniqueness constraint defined at the Generic level.
-
-```yaml {7}  showLineNumbers
-nodes:
-  - name: Person
-    namespace: Example
+```yaml
+generics:
+  - name: GenericDevice
+    namespace: Infra
+    description: "Generic Device object."
+    label: "Device"
+    icon: "mdi:server"
+    default_filter: name__value
+    order_by:
+      - name__value
+    display_labels:
+      - name__value
     attributes:
       - name: name
         kind: Text
         unique: true
+        order_weight: 1000
+      - name: description
+        kind: Text
+        optional: true
+        order_weight: 1100
+
+nodes:
+  - name: Device
+    namespace: Infra
+    inherit_from:
+      - "GenericDevice"
+    attributes:
+      - name: role
+        kind: Dropdown
+        optional: true
+        choices:
+          - name: core
```

Comment:
I think we could drop 2 or 3 choices from this list to keep it short and easier to read